### PR TITLE
refactor: Simplify Cryptobox initialization

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: src/script/cryptography/CryptographyRepository.js [hunk 1]
```diff
@@ -131,6 +77,11 @@ export class CryptographyRepository {
       const {userId, clientId} = ClientEntity.dismantleUserClientId(sessionId);
       amplify.publish(WebAppEvents.CLIENT.ADD, userId, {id: clientId}, true);
     });
+
+    if (useExistingCryptobox === true) {
```
Comment (lipis) [inline src/script/cryptography/CryptographyRepository.js]:
```suggestion
    if (useExistingCryptobox) {
```

File: src/script/main/app.js [hunk 2]
```diff
@@ -388,7 +388,7 @@ class App {
       telemetry.time_step(AppInitTimingsStep.VALIDATED_CLIENT);
       telemetry.add_statistic(AppInitStatisticsValue.CLIENT_TYPE, clientEntity.type);
 
-      await cryptographyRepository.loadCryptobox(this.service.storage.db);
+      await cryptographyRepository.initCryptobox(true);
```
Comment (lipis) [inline src/script/main/app.js]: default value is `true`
```suggestion
      await cryptographyRepository.initCryptobox();
```

File: test/helper/TestFactory.js [hunk 3]
```diff
@@ -118,11 +118,12 @@ export class TestFactory {
     this.cryptography_repository = new CryptographyRepository(this.cryptography_service, this.storage_repository);
     this.cryptography_repository.currentClient = ko.observable(currentClient);
 
-    if (mockCryptobox) {
+    if (mockCryptobox === true) {
```
Comment (lipis) [inline test/helper/TestFactory.js:121]: was that an issue?

File: src/script/main/app.js (hunk 2 above)
Comment (bennycode) [reply src/script/main/app.js]: I don't want to rely on default values. I am actually considering removing the default.

File: test/helper/TestFactory.js (hunk 3 above)
Comment (bennycode) [reply test/helper/TestFactory.js:121]: I want to be sure to check against a boolean. If `mockCryptobox` would be `'a'` the condition would still be valid but it should not.

File: src/script/cryptography/CryptographyRepository.js (hunk 1 above)
Comment (bennycode) [reply src/script/cryptography/CryptographyRepository.js]: `useExistingCryptobox` should be a boolean, I don't want to wrongly accept numbers or characters.

**Pull request merged**